setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup(
+    name='Embedded2',
+    version='0.5.0',
+    description='Fancy goggle detector',
+    author='CAM2',
+    author_email='cam2@purdue.edu',
```

Comment:
What is the purpose of providing this email? Is cam2@purdue.edu an email that someone monitors?

Reply:
these are placeholder values. I do not know the correct information. If you do could you please provide it so I can make that change?

Reply:
Make the version 1.0.0 I guess? We haven’t talked about versioning at all. Author could be ‘CAM2 Embedded Vision Team’. 
How about Akshay’s Purdue email? Or just don’t provide an email if it’s not necessary

Reply:
Again Also a placeholder the goal of this PR was to fix travis. Since this file wasn't setup I added because its needed to build the project. I think we should just defer completing this file until we have all this information (most likely something we should discuss at a meeting) and are closer to deploying.